gcc/config/riscv/riscv.md

```diff
@@ -154,7 +226,7 @@
 ;; logical      integer logical instructions
 ;; shift	integer shift instructions
 ;; slt		set less than instructions
-;; imul		integer multiply 
+;; imul		integer multiply
```

Comment:
Don't fix that this in this patch, maybe you could collect those clean up stuff to single patch and go upstream directly.

Reply:
I fix the issue here:https://github.com/riscv-collab/riscv-gcc/pull/334/commits/15ba09286c0af8f6e23a2f9ece7a7a716002556b. Feel free to comment it